state_trace.go

```diff
@@ -0,0 +1,342 @@
+// Copyright 2015 The etcd Authors
```

Comment:
Please fix the year (here and in other files).

Reply:
Done.